src/components/controls/TextInput/TextInput.scss

```diff
@@ -35,6 +35,25 @@
     @include mixins.text-accent;
 }
 
+@mixin input-control($size) {
```

Comment:
Add here some description please - why we use corrected mixin here

Reply:
fixed it